Sources/Filters/General/ContourTriangulator/index.d.ts

```diff
@@ -0,0 +1,117 @@
+import { vtkAlgorithm, vtkObject } from "../../../interfaces";
+import { Vector3 } from "../../../types";
+import vtkCellArray from "vtk.js/Sources/Common/DataModel/CellArray";
+import vtkPolyData from "vtk.js/Sources/Common/DataModel/PolyData";
+
+/**
+ *
+ */
+export interface IContourTriangulatorInitialValues {
+	triangulationErrorDisplay?: boolean;
+}
+
+type vtkContourTriangulatorBase = vtkObject & vtkAlgorithm;
+
+export interface vtkContourTriangulator extends vtkContourTriangulatorBase {
+	/**
+	 *
+	 * @param {any} inData
+	 * @param {any} outData
+	 */
+	requestData(inData: any, outData: any): void;
+
+	/**
+	 *
+	 */
+	triangulateContours(): boolean;
+
+	/**
+	 *
+	 */
+	triangulatePolygon(): boolean;
```

Comment:
But then why does every other `index.d.ts` have tabs?

Reply:
fair enough, leave it with tabs then. My bad

Reply:
I actually added extra rules for linting `index.d.ts` files to `.prettier.config.js` and thought we could add this in a separate MR, since I noticed that some of them aren't linted.

```js
module.exports = {
  printWidth: 80,
  singleQuote: true,
  trailingComma: 'es5',
  arrowParens: 'always',
  overrides: [
    {
      files: '*.d.ts',
      options: {
        useTabs: true,
        tabWidth: 4,
        singleQuote: false,
      },
    },
  ],
};
```

@daker What do you think?

Reply:
Good idea, but in a separate PR.
I don't understand why we should use tabs for d.ts files though... 
@floryst

Reply:
@DavidBerger98 yes it's discussed in this PR #2459